src/main/scala/uk/gov/hmrc/govukfrontend/views/implicits/RichTextareaSupport.scala

```diff
@@ -54,6 +56,22 @@ trait RichTextareaSupport {
         .withValue(field)
         .withHtmlErrorMessage(field)
 
+    // FIXME: PoC only - needs tests and functionality to not-override any existing label
+    def withHeadingAndSectionCaption(heading: Content, sectionCaption: Content): Textarea =
```

Comment:
Simplifies boilerplate needed to follow the 1-question per page pattern when using a section caption. Harder to do with implicit conversions due to the need for two parameters.